samples/rag-aisearch/csharp-inproc/host.json

```diff
@@ -0,0 +1,18 @@
+{
+  "version": "2.0",
+  "logging": {
+    "logLevel": {
+      "Microsoft.Azure.WebJobs.Extensions.OpenAI": "Information"
+    }
+  },
+  "extensions": {
+    "openai": {
+      "searchProvider": {
+        "type": "azureAiSearch",
+        "isSemanticSearchEnabled": "true",
+        "useSemanticCaptions": "true",
+        "vectorSearchDimensions": "1536"
```

Comment:
Now that you've changed the types of these fields in `AzureAISearchConfigOptions` from `string` to `int` or `bool`, should we update these host.json values to match? For example, using `true` instead of `"true"`, and using `1536` instead of `"1536"`?